sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare_async.py

```diff
@@ -294,7 +303,7 @@ async def test_out_of_order_ids(self, client):
             self.assertEqual(resp.id, expected_order[idx])
 
     @GlobalTextAnalyticsAccountPreparer()
-    @TextAnalyticsClientPreparer()
+    @TextAnalyticsClientPreparer(client_kwargs={"text_analytics_account_key": os.environ.get("AZURE_TEXT_ANALYTICS_KEY"), "text_analytics_account": os.environ.get("AZURE_TEXT_ANALYTICS_ENDPOINT")})
```

Comment:
hey @abhahn can you get rid of these environ variables in the future? i'm going to just push to this branch, but I think you adding this means your tests aren't fully set up correctly, can hop on a call today to figure this out more.

Reply:
Hey, looks like you did this already.  I have had some issues getting the tests to run correctly in WSL, so it might help for us to review what I'm doing to see if there's something wrong.